Gemfile

```diff
@@ -58,18 +55,18 @@ gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'statsd-instrument'
 gem 'twitter-text'
 gem 'tzinfo-data'
+gem 'webpacker', github: 'rails/webpacker'
```

Comment:
Yeah ... I think this was only possible once Rails 5.1 came out, and before that you DID need to pull from GH repo